src/packets/LoginUdp/LoginUdp_11/loginpackets.ts

```diff
@@ -23,7 +23,8 @@ const serverField: any[] = [
   { name: "descriptionId", type: "uint32" },
   { name: "reqFeatureId", type: "uint32" },
   { name: "serverInfo", type: "string" },
-  { name: "populationLevel", type: "uint32" },
+  { name: "populationNumber", type: "uint16", defaultValue: 0 },
```

Comment:
Should put a comment about this explaining that we do modify this packet and that it's not is correct structure